elements/rh-footer/rh-footer-links.ts

```diff
@@ -13,19 +12,28 @@ export class RhFooterLinks extends LitElement {
   static readonly styles = style;
 
   /**
-   * Visibily hide the header slot. Setting this to true will not affect
-   * aria-labelledby.
+   * Cause the header slot to be visually hidden.
+   * Setting this to true will not affect `aria-labelledby`.
    */
-  @property({ type: Boolean, attribute: 'header-hidden', reflect: true }) public headerHidden = false;
+  @property({
+    type: Boolean,
+    attribute: 'header-hidden',
+    reflect: true,
+  }) headerHidden = false;
 
-  private logger = new Logger(this);
+  #logger = new Logger(this);
 
-  protected slots = new SlotController(this, {
-    slots: ['header'],
-  });
+  #mo = new MutationObserver(() => this.updateAccessibility());
 
-  @initializer()
-  public updateAccessibility() {
+  protected slots = new SlotController(this, 'header');
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.updateAccessibility();
+    this.#mo.observe(this, { childList: true });
```

Comment:
Just verifying; do we need to observe the childList tree?

Reply:
this forwards the functionality from the `@initializer` decorator

https://github.com/patternfly/patternfly-elements/blob/898b61d8394ed39f2bb27f1e662a31726ccfa169/core/pfe-core/controllers/light-dom-controller.ts#L36-L45